### DISCUSSION TITLE
Your login session has expired

### DISCUSSION THREAD

**Discussion by rgcosta7** [dani-garcia/vaultwarden#2530] [Q&A]
> Hi guys,

I'm starting playing around with docker swarm.
I've all the latest images, docker, portainer, vaultwarden, traefik.

When I scale from 1 container to 2 or more I start getting this message as soon as I login:

Logged Out: Your session has expired

on the logs I couldn't find any error just the info that the user has login successful

[2022-06-03 18:52:27.001][vaultwarden::api::identity][INFO] User rxxxxxxxxxxx@gmail.com logged in successfully. IP: 10.0.0.2
[2022-06-03 18:52:27.001][response][INFO] (login) POST /identity/connect/token => 200 OK
[2022-06-03 18:52:26.771][request][INFO] POST /identity/connect/token

Is anyone else getting the same?

Any info/help is appreciated.

Here's my docker stack just:
[bitwarden-shared.txt](https://github.com/dani-garcia/vaultwarden/files/8835087/bitwarden-shared.txt)

Regards
R

**Comment by BlackDex** [comment]
I suggest to read https://vaultwarden.discourse.group/t/can-bitwarden-rs-tolerate-being-clustered/713/3?u=blackdex

In short it can, but there are issues with some features.
The reason you are being logged out is because the rsa key's are probably not the same.

**Comment by rgcosta7** [comment]
Hi BlackDex,

Thank you for your reply. 
At first I thought that I've miss the "The rsa key’s used for the JWT need to be shared." 

but I have a persist shared folder for the service and I'm using external mariadb that shouldn't be enough?!

I'll give a go again and try look in the logs

Cheers
R
